redwood-treehouse-composeui-insets/src/androidMain/kotlin/app/cash/redwood/treehouse/composeui/insets.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@file:JvmName("insetsAndroid")
+
+package app.cash.redwood.treehouse.composeui
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.asPaddingValues
+import androidx.compose.foundation.layout.systemBars
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.LocalLayoutDirection
+import androidx.compose.ui.unit.LayoutDirection
+import app.cash.redwood.layout.api.Margin
+
+@Composable
+public actual fun safeAreaInsets(): Margin {
+  val layoutDirection = LocalLayoutDirection.current
+  return WindowInsets.systemBars.asPaddingValues().toMargin(layoutDirection)
```

Comment:
Probably want to use `safeDrawing` insets here ([link](https://developer.android.com/reference/kotlin/androidx/compose/foundation/layout/package-summary#(androidx.compose.foundation.layout.WindowInsets.Companion).safeDrawing())). System bars only contains the nav/status bars, but we also need to handle IME and cutouts.